web-ui/src/components/notes/Note.jsx

```diff
@@ -86,12 +99,25 @@ const Notes = (props) => {
           Notes for {memberName}
         </h1>
         <div className="container">
-          <textarea
-            onChange={handleNoteChange}
-            value={note.description}
-          ></textarea>
-          <canvas ref={canvasRef}></canvas>
+          {isLoading ? (
+            <div className="is-loading"></div>
+          ) : (
+            <textarea
+              onChange={handleNoteChange}
+              value={note && note.description ? note.description : ""}
+            ></textarea>
+          )}
         </div>
+        <Snackbar
+          autoHideDuration={2500}
+          open={open}
+          onClose={handleClose}
+          style={{ left: "40%", bottom: "20%" }}
+        >
+          <Alert onClose={handleClose} severity="error">
+            {toast}
+          </Alert>
+        </Snackbar>
```

Comment:
Do we not already have a top level snackbar? If not, we should probably make one and have a means of enqueueing toast to it via the context.